packages/react-hooks/src/useBalancesAll.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2021 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { DeriveBalancesAll } from '@polkadot/api-derive/types';
+import { useApi, useCall } from '@polkadot/react-hooks';
+
+/**
+ * Gets the account full balance information
+ *
+ * @param accountAddress The account address of which balance is to be returned
+ * @returns full information about account's balances
+ */
+export function useBalancesAll (accountAddress: string): DeriveBalancesAll | undefined {
```

Comment:
Needs to be exported for the react-hook/index.ts

Reply:
Done